multiversx_sdk_cli/projects/shared.py

```diff
@@ -22,3 +28,29 @@ def _directory_contains_file(directory: Path, name_suffix: str) -> bool:
         if str(file).lower().endswith(name_suffix.lower()):
             return True
     return False
+
+
+def are_clang_and_cpp_dependencies_installed() -> bool:
```

Comment:
Wrt. the other comment, can directly throw error here.

Reply:
Throwing error here.